application/core/LimeMailer.php

```diff
@@ -1005,10 +1005,9 @@ public static function validateAddress($address, $patternselect = null)
 
     /**
      * Validate a list of email addresses - either as array or as comma or semicolon-limited text
-     * @return string List with valid email addresses - invalid email addresses are filtered - false if none of the email addresses are valid
-     * @param string $aEmailAddressList  Email address to check
+     * @param string|string[] $aEmailAddressList  Email address to check
      * @param string|callable $patternselect Which pattern to use (default to static::$validator)
-     * @returns array
+     * @return string List with valid email addresses - invalid email addresses are filtered - false if none of the email addresses are valid
```

Comment:
`string[]|false ` or  `array|false`

It's not a string list : it's an array of list